modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About ExternalIP
+
+For non-cloud environments, {product-title} supports the assignment of external IP addresses to a Service `spec.externalIPs` field through the *ExternalIP* facility.
+This exposes an additional virtual IP address, assigned to the Service, that can be outside the service network defined for the cluster.
+A Service configured with an external IP functions similarly to a Service with `type=NodePort`, allowing you to direct traffic to a local node for load balancing.
+
+You must configure your networking infrastructure to ensure that the external IP address blocks that you define are routed to the cluster.
+
+{product-title} extends the ExternalIP functionality in Kubernetes by adding the following capabilities:
+
+- Restrictions on the use of external IP addresses through a configurable policy
+- Allocation of an external IP address automatically to a Service upon request
+
+By default, only a user with `cluster-admin` privileges can create a Service with `spec.externalIPs[]` set to IP addresses defined within an external IP address block.
+
+[CAUTION]
+====
+Disabled by default, use of ExternalIP functionality can be a security risk, because in-cluster traffic to an external IP address is directed to that Service.
+This could allow cluster users to intercept sensitive traffic destined for external resources.
+====
+
+[IMPORTANT]
+====
+This feature is supported only in non-cloud deployments.
+For cloud deployments, use the load balancer services for automatic deployment of a cloud load balancer to target the endpoints of a service.
+====
+
+You can assign an external IP address in the following ways:
+
+Automatic assignment of an external IP::
+{product-title} automatically assigns an IP address from the `autoAssignCIDRs` CIDR block to the `spec.externalIPs[]` array when you create a Service CR object.
+In this case, {product-title} implements a non-cloud version of the load balancer Service type and assigns IP addresses to the services.
+
+Manual assignment of an external IP::
+{product-title} uses the IP addresses assigned to the `spec.externalIPs[]` array when you create a Service CR. You cannot specify an IP address that you have already used in another Service.
+
+[id="configuration-externalip_{context}"]
+== Configuration for ExternalIP
+
+Use of an external IP address in {product-title} is governed by the following fields:
```

Comment:
I would call out that this is in the `config.openshift.io/v1/Network` object.

Reply:
@squeed, oh, wow, somehow I never mention that important detail. How about

> Use of an external IP address in {product-title} is governed by the following fields in the `Network.config.openshift.io` CR named `cluster`: